### PR TITLE
feat: export user resume as .docx

### LINKED ISSUE #415: allow admin to export the user experiences in docx format
gem to use https://github.com/ruby-docx/docx

### PR DESCRIPTION
**Pull request by gabrielcosta42**
Generates a .docx document to export the user experiences tab

Adding 2 .docx to the codebase:
- template.docx is the structure of the final docment
- template-elements.docx which holds the elements to mount the doc

closes #415

### CODE REVIEW

File: app/admin/user.rb [hunk 1]
```diff
@@ -335,4 +339,28 @@ def create
...
+    doc.mount_professional_section(job_experiences)
+
+    education_experiences = user.education_experiences.decorate
+    doc.mount_education_section(education_experiences)
+
+    contribution_experiences = user.contributions.approved.order(created_at: :asc).decorate
+    doc.mount_contribution_section(contribution_experiences)
+
+    talk_experiences = user.talks.decorate
+    doc.mount_talk_section(talk_experiences)
```
Comment (adbatista) [inline app/admin/user.rb:362]: this could be in a external class

File: app/decorators/professional_experience_decorator.rb [hunk 2]
```diff
@@ -8,6 +8,17 @@ def start_date
   end
 
   def end_date
-    model.start_date.to_date.to_fs(:date)
+    return I18n.t(:present) if model.end_date.nil?
+    model.end_date.to_date.to_fs(:date)
+  end
+
+  def start_date_month
+    return I18n.t(:present) if model.start_date.nil?
```
Comment (adbatista) [inline app/decorators/professional_experience_decorator.rb:16]: if has no start date it did not start, so should not have this record

Comment (gabrielcosta42) [reply app/decorators/professional_experience_decorator.rb]: that's true. solved.

File: app/admin/user.rb (hunk 1 above)
Comment (gabrielcosta42) [reply app/admin/user.rb]: should it also go in the `UserResumeDocService`?

Comment (adbatista) [reply app/admin/user.rb]: no, UserResumeDoc, it should not be a service, but ok.
User resume doc describes the word doc, and we should have a class that builds the doc

Comment (gabrielcosta42) [reply app/admin/user.rb]: done

File: Gemfile [hunk 3]
```diff
@@ -38,14 +38,16 @@ gem 'kaminari'
...
+
 gem 'httparty'
 gem 'github_api', '~> 0.18.2'
 gem 'active_model_serializers', '~> 0.10.13'
 gem 'money-rails', '~> 1.12'
 
 gem 'ransack', '~> 2.3'
 
-gem 'sidekiq', "< 7"
+gem 'sidekiq', "< 8"
```
Comment (adbatista) [inline Gemfile:50]: this is not necessary now and redis on heroku does not support yet

**Review by adbatista — APPROVED** [review]

File: Gemfile (hunk 3 above)
Comment (gabrielcosta42) [reply Gemfile:50]: ok, removing it.

**Pull request merged**